site/content/how-to/traffic-management/response-headers.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "HTTP Response Headers"
+description: "Learn how to modify the response headers of your application using NGINX Gateway Fabric."
+weight: 700
+toc: true
+---
+
+[HTTP Header Modifiers](https://gateway-api.sigs.k8s.io/guides/http-header-modifier/?h=request#http-header-modifiers) can be used to add, modify or remove headers during the request-response lifecycle. The [ResponseHeaderModifier](https://gateway-api.sigs.k8s.io/guides/http-header-modifier/#http-response-header-modifier) is used to alter headers in a response to the client.
+
+In this guide we will modify the headers for HTTP responses when client requests are made. For an introduction to exposing your application, we recommend that you follow the [basic guide]({{< relref "/how-to/traffic-management/routing-traffic-to-your-app.md" >}}) first.
+
+
+## Prerequisites
+
+- [Install]({{< relref "/installation/" >}}) NGINX Gateway Fabric.
+- [Expose NGINX Gateway Fabric]({{< relref "installation/expose-nginx-gateway-fabric.md" >}}) and save the public IP
+  address and port of NGINX Gateway Fabric into shell variables:
+
+   ```text
+   GW_IP=XXX.YYY.ZZZ.III
+   GW_PORT=<port number>
+   ```
+
+{{< note >}}In a production environment, you should have a DNS record for the external IP address that is exposed, and it should refer to the hostname that the gateway will forward for.{{< /note >}}
+
+
+## Response Header Filter
+
+In this guide, we'll begin by configuring an app with custom headers and a straightforward httproute. We'll then observe the server response in relation to header responses. Next, we'll delve into modifying some of those headers using an httpRoute with filters to modify *response* headers. Our aim will be to verify whether the server responds with the modified headers.
+
+### Deploy the Headers application
+
+Begin by deploying the example application `headers`. It is a simple application that adds response headers which we'll later tweak and customize.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: headers
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: headers
+```
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.3.0/examples/http-response-header-filter/headers.yaml
+```
+
+Verify if the Pod is running in the `default` Namespace:
```

Comment:
Let's change this to something like: 

This will create the headers service and a deployment with one pod. Run the following command to verify the resources were created:

`kubectl get pods,svc`